### ISSUE TITLE
installing error in install_github

### ISSUE THREAD

**Issue by ykunisato** [yxlin/ggdmc#1]
When I tried to install ggdmc, I got following error message.

> devtools::install_github("yxlin/ggdmc")
Downloading GitHub repo yxlin/ggdmc@master
Error in FUN(X[[i]], ...) : 
  Invalid comparison operator in dependency: >= 

Installing from source is working. Thank you for a great R package.

**Comment by yxlin** [comment]
Yoshihiko, 

Thank you for reporting this bug.  I've looked into it. The bug appears from the trailing spaces in the DESCRIPTION file.  I have cleaned them up and tested on my Windows machine.  I hope it now also work on your machine.

**Comment by ykunisato** [comment]
Yi-Shin,

Thanks!
Now, installing process is started. But, it fails in testing.

###############################################
devtools::install_github("yxlin/ggdmc")
Downloading GitHub repo yxlin/ggdmc@master

...

-lm -lquadmath -lasan -lubsan -L/usr/local/lib/R/lib -lR
installing to /usr/local/lib/R/site-library/ggdmc/libs
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
==4509==ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
ERROR: loading failed
 removing ‘/usr/local/lib/R/site-library/ggdmc’
Error in i.p(...) : 
  (converted from warning) installation of package ‘/tmp/RtmpxMRYMW/filecd83a05d307/ggdmc_0.2.5.9.tar.gz’ had non-zero exit status
###############################################

**Comment by yxlin** [comment]
My apology and I appreciate letting me know this.  I forgot to remove the two C++ flags  -lasan -lubsan in Makevars.in and uploaded it.  They are testing flags and should not be there.  I am in the process of fixing memory leakage in FCalculator.cpp, so use them.  I  have removed them in this update.  I tested on my Ubuntu, so hopefully it now works on your machine, too.

**Comment by ykunisato** [comment]
It is working well now (on rocker/verse). Thank you so much!

**Comment by Joe-zhang-13** [comment]
C:/RBuildTools/3.5/mingw_32/bin/g++  -std=gnu++11 -I"C:/PROGRA~1/R/R-36~1.3/include" -DNDEBUG  -I"C:/Users/HP/Documents/R/win-library/3.6/Rcpp/include" -I"C:/Users/HP/Documents/R/win-library/3.6/RcppArmadillo/include"     -I../inst/include -I. -fopenmp   -O2 -Wall  -mtune=core2 -c Prior.cpp -o Prior.o
Prior.cpp:17:29: fatal error: gsl/gsl_randist.h: No such file or directory
 #include <gsl/gsl_randist.h>
                             ^
compilation terminated.
make: *** [C:/PROGRA~1/R/R-36~1.3/etc/i386/Makeconf:215: Prior.o] Error 1
ERROR: compilation failed for package 'ggdmc'
* removing 'C:/Users/HP/Documents/R/win-library/3.6/ggdmc'
* restoring previous 'C:/Users/HP/Documents/R/win-library/3.6/ggdmc'
错误: Failed to install 'ggdmc' from GitHub:
  (由警告转换成)installation of package ‘C:/Users/HP/AppData/Local/Temp/RtmpU3pazO/file338c3620541c/ggdmc_0.2.8.0.tar.gz’ had non-zero exit status
I don't know how to fix this error. I check online. However, there is no satisfying answer about this error

**Comment by yxlin** [comment]
Hi Joe,

Thank you so much for reporting the error. What I could see from the error message is it was caused by the [GSL library](https://www.gnu.org/software/gsl/), which unfortunately is not native in Windows system.  Although it is possible to install GSL on Windows, It would involve a lot of manually tweaking and modify search path. I was testing random number generations using  a  bivariate gaussian PDF, forgot to remove it afterward and accidentally uploaded the testing codes, so the error.  I apologize for the confusion.  I have commented out the part using gsl and have tested it again on my Windows OS.  I think the version of 0.2.8.0 should install on Windows as normal.   

Best,

Yi-Shin
